sbol/property.py

```diff
@@ -741,9 +741,15 @@ def _to_uri(self, obj):
         raise TypeError(errmsg.format(obj, type(obj)))
 
     def setSinglePropertyValue(self, new_value):
-        self._sbol_owner.properties[self._rdf_type] = [self._to_uri(new_value)]
+        if new_value is None or new_value == '':
+            self._sbol_owner.properties[self._rdf_type] = []
+        if new_value:
```

Comment:
This one line is going to lead to unpleasant surprise for some programmer in the future. This will cause the method to ignore things like `False`, `[]`, and `{}`. So a user could set a new value, and then discover that their request to set the attribute has been silently ignored. Better to let `_to_uri()` take care of things. It either converts a value to a URI or raises an exception. Nothing false through the cracks.

I would use a `return` on the first if, and then set to the new value on the last line and let `_to_uri()` take care of bad values.

Reply:
Done